app/templates/buyers/delete_brief.html

```diff
@@ -0,0 +1,71 @@
+{% extends "_base_page.html" %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block pageTitle %}
+  {{ brief.title or brief.lotName }} – Digital Marketplace
```

Comment:
The page title should be the same as/similar to the page breadcrumb

Reply:
Done